### PR TITLE
template for cm

### PR DESCRIPTION
**Pull request by ArielLima**
Template for the ConfigMap used by this operator

### CODE REVIEW

**Comment by maorfr** [comment]
this should go in to this template IMO:
https://github.com/openshift/aws-account-operator/blob/master/hack/olm-registry/olm-artifacts-template.yaml

File: deploy/configurations.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+metadata:
+  name: aws-account-operator-configmap
+objects:
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: aws-account-operator-configmap
+    namespace: aws-account-operator
+  data:
+    account-limit: ${ACCOUNT_LIMIT}
```
Comment (maorfr) [inline deploy/configurations.yaml:19]: I would add quotes around `${ACCOUNT_LIMIT}`

Comment (ArielLima) [reply deploy/configurations.yaml:19]: By adding the quote here do I remove them from the app-interface pr? -> https://gitlab.cee.redhat.com/service/app-interface/-/blob/d3f2ce5db392eb9e55d5ddeeee58a70d986034d2/data/services/osd-operators/cicd/saas/saas-aws-account-operator.yaml#L42-42

**Comment by ArielLima** [comment]
> this should go in to this template IMO:
> https://github.com/openshift/aws-account-operator/blob/master/hack/olm-registry/olm-artifacts-template.yaml

Sure

File: deploy/configurations.yaml (hunk 1 above)
Comment (maorfr) [reply deploy/configurations.yaml]: nope, keep on both

**Comment by fahlmant** [comment]
@rogbas @fahlmant please review

**Comment by ArielLima** [comment]
@rogbas @fahlmant https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/6309

**Comment by rogbas** [comment]
The way it's set in the app-interface MR linked above, we have account limit == 100 in all environments. It doesn't make sense considering what we have today.

**Comment by rogbas** [comment]
@ArielLima can you please review the app-interface cicd config, making sure we have real account limits and not 100?

**Pull request merged**